clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/wgscram/CramMetadata.scala

```diff
@@ -87,7 +86,98 @@ case class CramMetadata(
+      ),
+      crosscheckPath = pathMapper(crosscheckPath, CramExtensions.CrossCheckExtension),
+      duplicateMetricsPath = pathMapper(
+        duplicateMetricsPath,
+        CramExtensions.DuplicateMetricsExtension
+      ),
+      fingerprintPath =
+        pathMapper(fingerprintPath, CramExtensions.FingerprintVcfExtension),
+      fingerprintVcfPath =
+        pathMapper(fingerprintVcfPath, CramExtensions.FingerprintVcfExtension),
+      fingerprintingSummaryMetricsPath = pathMapper(
+        fingerprintingSummaryMetricsPath,
+        CramExtensions.FingerprintingSummaryMetricsExtension
+      ),
+      fingerprintingDetailMetricsPath = pathMapper(
+        fingerprintingDetailMetricsPath,
+        CramExtensions.FingerprintingDetailMetricsExtension
+      ),
+      gcBiasPdfPath = pathMapper(gcBiasPdfPath, CramExtensions.GcBiasPdfExtension),
+      gcBiasSummaryMetricsPath =
+        pathMapper(gcBiasPdfPath, CramExtensions.GcBiasSummaryMetricsExtension),
+      gcBiasDetailMetricsPath =
+        pathMapper(gcBiasPdfPath, CramExtensions.GcBiasDetailMetricsExtension),
+      insertSizeHistogramPath = pathMapper(
+        insertSizeHistogramPath,
+        CramExtensions.InsertSizeHistogramPdfExtension
+      ),
+      insertSizeMetricsPath =
+        pathMapper(insertSizeMetricsPath, CramExtensions.InsertSizeMetricsExtension),
+      qualityDistributionPdfPath = pathMapper(
+        qualityDistributionPdfPath,
+        CramExtensions.QualityDistributionPdfExtension
+      ),
+      qualityDistributionMetricsPath = pathMapper(
+        qualityDistributionMetricsPath,
+        CramExtensions.QualityDistributionMetricsExtension
+      ),
+      rawWgsMetricsPath =
+        pathMapper(rawWgsMetricsPath, CramExtensions.RawWgsMetricsExtension),
+      readgroupAlignmentSummaryMetricsPath = pathMapper(
+        readgroupAlignmentSummaryMetricsPath,
+        CramExtensions.ReadGroupAlignmentMetricsExtension
+      ),
+      readgroupGcBiasPdfPath =
+        pathMapper(readgroupGcBiasPdfPath, CramExtensions.ReadGroupGcBiasPdfExtension),
+      readgroupGcBiasSummaryMetricsPath = pathMapper(
+        readgroupGcBiasSummaryMetricsPath,
+        CramExtensions.ReadGroupGcBiasSummaryMetricsExtension
+      ),
+      readgroupGcBiasDetailMetricsPath = pathMapper(
+        readgroupGcBiasDetailMetricsPath,
+        CramExtensions.ReadGroupGcBiasDetailMetricsExtension
+      ),
+      recalDataPath = pathMapper(recalDataPath, CramExtensions.RecalDataExtension),
+      baitBiasSummaryMetricsPath = pathMapper(
+        baitBiasSummaryMetricsPath,
+        CramExtensions.BaitBiasSummaryMetricsExtension
+      ),
+      baitBiasDetailMetricsPath = pathMapper(
+        baitBiasDetailMetricsPath,
+        CramExtensions.BaitBiasDetailMetricsExension
+      ),
+      wgsMetricsPath = pathMapper(wgsMetricsPath, CramExtensions.WgsMetricsExension)
+//      readgroupLevelMetricsFiles = pathMapper(readgroupLevelMetricsFiles)
```

Comment:
Test I asked for will make sure this gets uncommented.

I think you want something like:
```scala
readgroupLevelMetricsFiles = readgroupLevelMetricsFiles.map(pathMapper(_, ???))
```
Though as I typed it I realized there's no consistent extension to use...

Reply:
Something like this might work:
```scala
readgroupLevelMetricsFiles = readgroupLevelMetricsFiles.map { path =>
  val name = new File(path).getName
  pathMapper(path, name)
}
```
It's a bad hack, but I think it'll fool the existing code. Better would be for us to restructure this logic to distinguish things-which-can-change-their-base-name from things-which-can-only-be-moved, but that's for another ticket.